locations/spiders/mol.py

```diff
@@ -0,0 +1,183 @@
+import scrapy
+from scrapy.http import FormRequest
+
+from locations.categories import Categories, Extras, Fuel, FuelCards, PaymentMethods, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.geo import country_coordinates
+from locations.hours import NAMED_DAY_RANGES_EN, OpeningHours
+
+BRANDS_MAPPING = {
+    "INA": ("INA", "Q1662137"),
+    "MOL": ("MOL", "Q549181"),
+    "MOL (AGIP)": ("MOL", "Q549181"),
+    "MOL Ceska Republika": ("MOL", "Q549181"),
+    "MOLPLUGEE": ("MOL", "Q549181"),
+    "PapOil": ("PapOil", None),
+    "Slovnaft": ("Slovnaft", "Q1587563"),
+}
+
+FUEL_MAPPING = {
+    "AdBlue": Fuel.ADBLUE,
+    "CNG (Comprimed natural gas, in tank)": None,
+    "EVO 100 Plus (GASOLINE PREMIUM)": Fuel.OCTANE_100,
+    "EVO 95": Fuel.OCTANE_95,
+    "EVO Diesel": Fuel.DIESEL,
+    "EVO Diesel Plus (DIESEL PREMIUM)": Fuel.DIESEL,
+    "LPG ": Fuel.LPG,
+    "MOL Racing Fuel": "fuel:octane_102",
+    "Maingrade 95": Fuel.OCTANE_95,
+    "Premium Gasoline": Fuel.OCTANE_95,
+    "Premium Diesel": Fuel.DIESEL,
+    "Maingrade Diesel": Fuel.DIESEL,
+    "RACING 102": "fuel:octane_102",
+    "XXL Diesel": Fuel.DIESEL,
+}
+
+SERVICES_MAPPING = {
+    "Air/Compresor": Extras.COMPRESSED_AIR,
+    "Automatic car wash": Extras.CAR_WASH,
+    "Big enough to fit a truck": "capacity:hgv",
+    "Disabled toilet": Extras.TOILETS_WHEELCHAIR,
+    "Family Toilet": Extras.TOILETS,
+    "Hamburger": "fast_food",
+    "Hoover": Extras.VACUUM_CLEANER,
+    "Hot dog": "fast_food",
+    "Internet (wifi)": Extras.WIFI,
+    "Lubricant": Fuel.ENGINE_OIL,
+    "Shower room": Extras.SHOWERS,
+    "Truck friendly": "capacity:hgv",
```

Comment:
I'm pretty sure this is better as an access attribute (hgv=yes)

Reply:
Thanks. Resolved.